### DISCUSSION TITLE
How to update only specific packages?

### DISCUSSION THREAD

**Discussion by belek** [renovatebot/renovate#28147] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

BitBucket Cloud

### Please tell us more about your question or problem

I would like to configure Renovate to create PRs only for specific packages.
I tried to use this configs in both renovate.json and config.js:
`"packageRules": [
      {
        "matchPackageNames": ["pydantic"]
      }
    ]`
But I got PRs for all outdated packages.

### Logs (if relevant)

<details><summary>Logs</summary>

```

DEBUG: Combined config
       "config": {
         "platform": "bitbucket",
         "gitAuthor": "Renovate Bot <[******](mailto:******)>",
         "username": "belek",
         "password": "***********",
         "onboarding": true,
         "onboardingConfig": {"extends": ["config:recommended"]},
         "recreateWhen": "always",
         "autodiscover": true,
         "autodiscoverFilter": [
           "belek_abylov/some-python-app",
           "belek_abylov/anotherrepo"
         ],
         "packageRules": [{"matchPackageNames": ["pydantic"]}],
         "hostRules": [
           {"hostType": "github", "matchHost": "[github.com](http://github.com/)", "token": "***********"}
         ]
       }
```

</details>

**Comment by secustor** [comment, marked as answer]
Your are matching `pydantic`, tough you are not changing any settings. 

Disable all updates except `pydantic`:
```json
{
  "packageRules": [
    "excludePackageNames": ["pydantic"],
    "enabled": false
  ]
}
```

**Reply by belek** [reply]
Thanks, I though `exludePackageNames` means exclude from updating specified packages.
